enum.go

```diff
@@ -868,16 +972,20 @@ var AllToolCategory = []string{
 type UserRole string
 
 const (
-	UserRoleUser      UserRole = "user"       // A regular user on the account.
-	UserRoleAdmin     UserRole = "admin"      // An administrator on the account.
-	UserRoleBasicUser UserRole = "basic_user" // A basic user on the account with limited access.
+	UserRoleUser           UserRole = "user"            // A regular user on the account.
+	UserRoleAdmin          UserRole = "admin"           // An administrator on the account.
+	UserRoleBasicUser      UserRole = "basic_user"      // A basic user on the account with limited access.
```

Comment:
`basic_user` is no more - why is it still here?

Reply:
Good question, I'll remove it